packages/experiments-realm/BlogPost/ultimate-guide-remote-work.json

```diff
@@ -29,29 +29,14 @@
           "self": "../BlogApp/ramped"
         }
       },
-      "categories.0": {
```

Comment:
Was the .0 .1 thing an old format?

Reply:
if value is null, it's `categories` only

Reply:
Roger